src/handleAction.js

```diff
@@ -1,19 +1,18 @@
 import isFunction from 'lodash/isFunction';
 import identity from 'lodash/identity';
 import isNil from 'lodash/isNil';
-import isSymbol from 'lodash/isSymbol';
+import includes from 'lodash/includes';
+import { ACTION_TYPE_DELIMITER } from './combineActions';
 
-export default function handleAction(type, reducers, defaultState) {
-  const typeValue = isSymbol(type)
-    ? type
-    : type.toString();
+export default function handleAction(actionType, reducers, defaultState) {
+  const actionTypes = actionType.toString().split(ACTION_TYPE_DELIMITER);
```

Comment:
It's used to keep compatibility with Symbol support; otherwise [this test](https://github.com/acdlite/redux-actions/blob/812d5f208f941c181b716bdb9fe524235ae51e96/src/__tests__/handleActions-test.js#L26) and another Symbol-related test fails.